xen/include/public/io/displif.h

```diff
@@ -494,6 +504,7 @@
  *   buffer size (buffer_sz) exceeds what can be addressed by this single page,
  *   then reference to the next page must be supplied (see gref_dir_next_page
  *   below)
+ * data_ofs - uint32_t, offset of the data in the buffer, octets
```

Comment:
Are there any requirements for the data_ofs itself (alignment, min/max value, etc)?

Reply:
well, it should be HW dependent, so probably we can't know, thus no reason for any limitations here